doc/source/computation.rst

```diff
@@ -288,46 +316,53 @@ compute the mean absolute deviation on a rolling basis:
 
    mad = lambda x: np.fabs(x - x.mean()).mean()
    @savefig rolling_apply_ex.png
-   pd.rolling_apply(ts, 60, mad).plot(style='k')
+   s.rolling(window=60).apply(mad).plot(style='k')
+
+.. _stats.rolling_window:
 
-The :func:`rolling_window` function performs a generic rolling window computation
+Rolling Windows
+~~~~~~~~~~~~~~~
+
+The :meth:`~Window.mean`, and :meth:`~Window.sum` functions perform a generic rolling window computation
 on the input data. The weights used in the window are specified by the ``win_type``
 keyword. The list of recognized types are:
 
-    - ``boxcar``
-    - ``triang``
-    - ``blackman``
-    - ``hamming``
-    - ``bartlett``
-    - ``parzen``
-    - ``bohman``
-    - ``blackmanharris``
-    - ``nuttall``
-    - ``barthann``
-    - ``kaiser`` (needs beta)
-    - ``gaussian`` (needs std)
-    - ``general_gaussian`` (needs power, width)
-    - ``slepian`` (needs width).
+- ``boxcar``
+- ``triang``
+- ``blackman``
+- ``hamming``
+- ``bartlett``
+- ``parzen``
+- ``bohman``
+- ``blackmanharris``
+- ``nuttall``
+- ``barthann``
+- ``kaiser`` (needs beta)
+- ``gaussian`` (needs std)
+- ``general_gaussian`` (needs power, width)
+- ``slepian`` (needs width).
 
 .. ipython:: python
 
    ser = pd.Series(np.random.randn(10), index=pd.date_range('1/1/2000', periods=10))
 
-   pd.rolling_window(ser, 5, 'triang')
+   ser.rolling(window=5, win_type='triang').mean()
 
-Note that the ``boxcar`` window is equivalent to :func:`rolling_mean`.
+Note that the ``boxcar`` window is equivalent to :meth:`~Rolling.mean`.
 
 .. ipython:: python
 
-   pd.rolling_window(ser, 5, 'boxcar')
-
-   pd.rolling_mean(ser, 5)
+   ser.rolling(window=5, win_type='boxcar').mean()
+   ser.rolling(window=5).mean()
 
 For some windowing functions, additional parameters must be specified:
 
 .. ipython:: python
 
-   pd.rolling_window(ser, 5, 'gaussian', std=0.1)
+   ser.rolling(window=5, win_type='gaussian').mean(std=0.1)
```

Comment:
Could we move these parameters inside the `rolling` call instead of having them in the `mean` method? That makes more sense to me (e.g., could add `**kwargs`).